var/spack/repos/builtin/packages/py-fparser/package.py

```diff
@@ -8,23 +8,27 @@
 
 class PyFparser(PythonPackage):
     """
-    This project is based upon the Fortran (77..2003) parser
+    This project is based upon the Fortran parser
     originally developed by Pearu Peterson for the F2PY project,
     www.f2py.com. It provides a parser for Fortran source code
-    implemented purely in Python with minimal dependencies.
+    (up to and including F2008) implemented purely in Python with
+    minimal dependencies.
 
     """
 
     # Links
     homepage = "https://github.com/stfc/fparser"
     git = "https://github.com/stfc/fparser.git"
-    pypi = "fparser/fparser-0.1.4.tar.gz"
+    pypi = "fparser/fparser-0.2.0.tar.gz"
+
+    maintainers = ["arporter"]
```

Comment:
Use directive syntax.
```suggestion
    maintainers("arporter")
```

Reply:
Done.